build.gradle.kts

```diff
@@ -10,16 +10,46 @@ buildscript {
   }
 
   dependencies {
-    classpath(Dependencies.kotlinAllOpenPlugin)
-    classpath(Dependencies.kotlinGradlePlugin)
-    classpath(Dependencies.dokkaGradlePlugin)
-    classpath(Dependencies.kotlinNoArgPlugin)
-    classpath(Dependencies.junitGradlePlugin)
-    classpath(Dependencies.mavenPublishGradlePlugin)
-    classpath(Dependencies.protobufGradlePlugin)
-    classpath(Dependencies.jgit)
-    classpath(Dependencies.wireGradlePlugin)
-    classpath(Dependencies.kotlinBinaryCompatibilityPlugin)
+    classpath(Dependencies.kotlinAllOpenPlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.kotlinGradlePlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.dokkaGradlePlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.kotlinNoArgPlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.junitGradlePlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.mavenPublishGradlePlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.protobufGradlePlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.jgit) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.wireGradlePlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.kotlinBinaryCompatibilityPlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
```

Comment:
What's this part about?

Reply:
Hrm, I don't recall adding that

Reply:
Oh wait - will fix.

Reply:
This was an attempt to find what plugin was pulling in log4j.